crates/block-producer/src/mempool/inflight_state/mod.rs

```diff
@@ -106,7 +106,9 @@ impl InflightState {
     ) -> Result<BTreeSet<TransactionId>, AddTransactionError> {
         // Separate verification and state mutation so that a rejected transaction
         // does not impact the state (atomicity).
-        self.verify_transaction(tx)?;
+        self.verify_transaction(tx).inspect_err(|e| {
+            tracing::warn!("Error on auth'd tx submission: {e:?}");
```

Comment:
Just checking we don't want error! instead

Reply:
I agree that `warn!` is a better fit here, but I would suggest emitting an event instead:

```rust
tracing::warn!(event = %"VerifyTransactionFailed", op = %"AddTransaction, error = ?e);
```

Then, it will be easier to quantify how frequently this failure occurs.